source/03-specifications/02-formats-for-counter-reports.rst

```diff
@@ -60,11 +60,11 @@ Table 3.f (below): COUNTER Report Header Elements
      - Example
 
    * - Report_Name
-     - The name of the report as it appears in :numref:`reports-for-libraries`.
+     - The name of the report as it appears in :numref:`reports`.
```

Comment:
Was the change of the reference from Section 3.1 to Section 4 (reports instead of reports-for-consumers) intentional?